public/main.js

```diff
@@ -138,4 +162,9 @@ function completeTodo(todo) {
     createRequest.send(JSON.stringify({isComplete: !todo.isComplete}));
 }
 
+function setFilter(filterType) {
+    filter = filterType;
+    reloadTodoList();
+}
+
 reloadTodoList();
```

Comment:
You don't need this `reloadTodoList` now that you're setting the filter at the top